README.md

```diff
@@ -93,6 +93,11 @@ Running this demo uses the config file to generate a `trainer/` module that is
 compatible with CAIP. It also generates `orchestration/pipeline.py`, which
 compiles a Kubeflow Pipelines pipeline.
 
+> Note: If you're using a GKE cluster without Workload Identity configured, the 
```

Comment:
I believe `>` is for quotes. Maybe use `_italics_` or `**emphasis**` instead?

I suggest changing the last sentence to
`If this occurs, you will need to update the your config.yaml with the new Kubeflow Pipelines URL and rerun the demo.`

Reply:
Ack, done!